Interfaces/API/Dirac.py

```diff
@@ -1826,6 +1727,8 @@ def getOutputSandbox( self, jobID, outputDir = None, oversized = True, noJobDir
 
   #############################################################################
   def delete( self, jobID ):
+    return self.deleteJob( jobID )
```

Comment:
May I recommend printing a warning here that this 'delete' method will be deprecated soon ?